core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLRAXRepresentationSuite.scala

```diff
@@ -0,0 +1,1387 @@
+   }
+
+  """
+
+
+  val badRepJSON3 =
+    """
+   {
+       "xml" : "<user xmlns='test.org/2'><json>{ &quot;lastName&quot; : &quot;Williams&quot;, &quot;firstName&quot; : &quot;Jorge&quot; }</json></user>"
+   }
+
+  """
+
+  val badRepJSON4 =
+    """
+   {
+       "firstName" : "Jorge",
+       "xml" : "<user xmlns='test.org/2'><json>{ &quot;firstName&quot; : &quot;Jorge&quot; }</json></user>"
+   }
+
+  """
+
+  val badRepJSON5 =
+    """
+   {
+       "firstName" : "Jorge",
+       "lastName"  : "Williams",
+       "wooga"     : false
+   }
+  """
+
+  val badRepJSON6 =
+    """
+   {
+       "firstName" : "Jorge",
+       "xml" : "<user xmlns='test.org/2'><json>{ &quot;lastName&quot; : &quot;Williams&quot;, &quot;firstName&quot; : &quot;Jorge&quot; }</json></user>",
+       "wooga" : true
+   }
+
+  """
+
+  val badRepJSON7 =
+    """
+   {
+       "firstName" : "Jorge",
+       "xml" : "<user xmlns='test.org/2' wooga='true'><json>{ &quot;lastName&quot; : &quot;Williams&quot;, &quot;firstName&quot; : &quot;Jorge&quot; }</json></user>"
+   }
+
+  """
+
+  val badRepJSON8 =
+    """
+   {
+       "firstName" : "Jorge",
+       "xml" : "<user xmlns='test.org/2'><json>{ &quot;item&quot; : false, &quot;lastName&quot; : &quot;Williams&quot;, &quot;firstName&quot; : &quot;Jorge&quot; }</json></user>"
+   }
+
+  """
+
+  val badRepJSON9 =
+    """
+   {
+       "firstName" : "Jorge",
+       "lastName" : "Williams",
+       "items" : [false, false, true]
+   }
+
+  """
+
+
+
+  val badRepXML1 =
+   <some_xml xmlns="test.org">
+       <json> /booga\  </json>
+  </some_xml>
+
+  val badRepXML2 =
+   <some_xml xmlns="test.org">
+       <json> 42 </json>
+    </some_xml>
+
+  val badRepXML3 =
+   <user xmlns="test.org" firstN="Jorge" lastName="Williams"/>
+
+  val badRepXML4 =
+    <user2 xmlns="test.org" fn="Jorge" lastN="Williams"/>
+
+  val badRepXML5 =
+   <some_xml xmlns="test.org" wooga="true">
+       <json> {
+         goodRepJSON2
+      }</json>
+    </some_xml>
+
+  val badRepXML6 =
+   <some_xml xmlns="test.org">
+       <json> {badRepJSON5} </json>
+  </some_xml>
+
+  val badRepXML7 =
+    <user2 xmlns="test.org" fn="Jorge" ln="Williams" firstName="Jorge"/>
+
+
+  val badRepXML8 =
+    <other_xml xmlns="test.org">
+      <xml>
+        (user :  firstName="Jorge" : lastName="Williams")
+      </xml>
+    </other_xml>
+
+  val badRepXML9 =
+    <other_xml xmlns="test.org">
+      <xml>
+        &lt;user xmlns="test.org" firstName="Jorge" last="Williams"/&gt;
+      </xml>
+    </other_xml>
+
+  val badRepXML10 =
+    <other_xml xmlns="test.org">
+      <xml>
+        &lt;user xmlns="test.org" firstName="Jorge" lastName="Williams" why="yes"/&gt;
+      </xml>
+    </other_xml>
+
+
+
+  //
+  //  Assertions!
+  //
+  def happyPathAssertions(validator : Validator, wadlDesc : String, configDesc : String) {
+    test(s"A PUT on /a should succeed with good XML 1 $wadlDesc with $configDesc") {
```

Comment:
All of these tests that are similar could be rolled up with a `Seq().foreach` like these:
 - https://github.com/rackerlabs/api-checker/blob/api-checker-2.5.1/core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala#L76-L89

Reply:
That roll up doesn't save a lot of code in these cases, but it does provide a grouping for similar tests.

Reply:
I like it, I'll refactor these and follow that pattern in the future.

Reply:
Actually, I think I'm going to skip this actual fix and save it for the next time I do this kind of test.  You're right that it doesn't save that much code, and I need to move on to the next PR.

Reply:
:+1: